modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -14,14 +14,14 @@ Amazon DynamoDB Connector v1.3
 
 
 ==== Parameters
-[%header,cols="20s,20a,30a,20a,10a"]
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
 | Connection a| * <<config_basic, Basic>> 
 * <<config_role, Role>> 
  | The connection types to provide to this configuration. | | x
-| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when it sees fit. |  | 
+| Expiration Policy a| <<ExpirationPolicy>> |  +++Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when it sees fit.+++ |  | 
```

Comment:
I wish we could reword "when it sees fit." If you can't think of a way to do that , we can ignore it for now.